lib/lita/adapters/slack/api.rb

```diff
@@ -55,6 +55,14 @@ def send_attachments(room_or_user, attachments)
           )
```

Comment:
You changed the permission of this file from 644 to 755

Reply:
😬 that was an accident! I'll fix that!

Reply:
Fixed! Thanks @rafaelfranca good eye.